docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[actions-and-connectors-api-create]]
+=== Create action API
+++++
+<titleabbrev>Create action API</titleabbrev>
+++++
+
+Create {kib} actions.
+
+[[actions-and-connectors-api-create-request]]
+==== Request
+
+`POST <kibana host>:<port>/api/actions/action`
+
+[[actions-and-connectors-api-create-request-body]]
+==== Request body
+
+`name`::
+  (Required, string) The display name for the action.
+
+`actionTypeId`::
+  (Required, string) The action type ID for the action.
+
+`config`::
+  (Required, object) The action configuration. Configuration properties vary depending on
+  the action type. Some action types will have no configuration properties. See the specific action type to determine required fields.
+
+`secrets`::
+  (Required, object) The secrets configuration for the action. Secrets properties vary
+  depending on the action type. Some action types will have no secrets properties. See the specific ation type to determine required fields.
+
+[[actions-and-connectors-api-create-request-codes]]
+==== Response code
+
+`200`::
+    Indicates a successful call.
+
+[[actions-and-connectors-api-create-example]]
+==== Example
+
+[source,sh]
+--------------------------------------------------
+$ curl -X POST api/actions/action  -H 'kbn-xsrf: true' -H 'Content-Type: application/json' -d '
+{
+  "name": "my-action",
+  "actionTypeId": ".index",
+  "config": {
+    "index": "test-index"
+  }
+}'
+--------------------------------------------------
+// KIBANA
+
+The API returns the following:
+
+[source,sh]
+--------------------------------------------------
+{
+  "id": "c55b6eb0-6bad-11eb-9f3b-611eebc6c3ad",
+  "actionTypeId": ".index",
+  "name": "my-action",
+  "config": {
+    "index": "test-index",
+    "refresh": false,
+    "executionTimeField": null
+  },
+  "isPreconfigured": false
+}
+--------------------------------------------------
+
+==== Action type configuration and secrets
```

Comment:
Since we link to <<action-types,Action and connector types>> above, I would remove lines 70-117.